packages/ui/.storybook/main.ts

```diff
@@ -55,6 +56,7 @@ const config: StorybookConfig = {
     getAbsolutePath('storybook-addon-react-router-v6'),
     getAbsolutePath('storybook-addon-sass-postcss')
   ],
+  staticDirs: ['../public'],
```

Comment:
if this file is just for tests, please put them in a 'tests' folder so they can be ignored from the docker image